install/caj/docs/CAJ2.1-ReleaseNotes.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=windows-1252">
+    <title>Jakarta Annotations TCK, Version 2.1 Release Notes, December 2021</title>
+    <!--
+    Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at    http://www.eclipse.org/legal/epl-2.0.    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,    version 2 with the GNU Classpath Exception, which is available at    https://www.gnu.org/software/classpath/license.html.
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0-->
+    <style type="text/css">
+<!--
+body {  background-color: #ffffdd; color: #000000}
+pre {  background-color: #eeeeee; color: #000099; font-size: smaller}
+code {  color: #000099; font-size: smaller}
+h1 {  color: #000099}
+h2 {  color: #000099}
+h3 {  color: #000099; text-decoration: underline}
+h4 {  font-style: italic; color: #000099}
+.tabletitle {  font-style: italic; color: #ffffcc; font-size: small; font-variant: small-caps}
+.subtitle {  font-style: italic; color: #ffffcc; font-variant: small-caps}
+.emphasize  {  font-style: italic; color: #000099}
+.step  { font-weight: bold}
+.note { background-color: #fffff0; border-bottom-width: thin; border-top-width: thin; border-top-style: inset; border-bottom-style: inset; font-size: -10%;}
+th {  background-color: #eeeeee; padding-left: 2pt; padding-right: 2pt;}
+-->
+</style></head>
+  <body>
+    <div align="center">
+      <h1>Jakarta Annotations Technology Compatibility Kit, Version 2.1<br>
+        <em class="emphasize">Release Notes, December 2021</em></h1>
+    </div>
+    <h2><a name="kit_contents">Kit Contents</a></h2>
+    <p>The Jakarta Annotations, Version 2.1 Technology Compatibility Kit (TCK)
+      includes the following items:</p>
+    <ul>
+      <li>
+        <p><strong>Jakarta Annotations TCK Signature Tests:</strong> Check that
+          all public APIs are supported and/or defined as specified in the
+          Jakarta Annotations specification, Version 2.1 implementation under
+          test.</p>
+      </li>
+    </ul>
+    <hr>
+    <h2><a name="platform">Platform Notes</a></h2>
+    <p>The Jakarta Annotations TCK tests have been built with JDK 11 and tested
+      with OpenJDK 11 and OpenJDK 17.</p>
+    <p>The Jakarta Annotations TCK tests have been run on the following
+      platforms:</p>
+    <ul>
+      <li>TBD: include OS + version</li>
```

Comment:
OS as we discussed at PR https://github.com/eclipse-ee4j/jakartaee-tck/pull/788#discussion_r781793098

Reply:
Switched OS to `Debian`